### DISCUSSION TITLE
/java8/optional.html

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#36] [Announcements]
# /java8/optional.html

一份通俗易懂、风趣幽默的Java学习指南，内容涵盖Java基础、Java并发编程、Java虚拟机、Java企业级开发、Java面试等核心知识点。学Java，就认准Java程序员进阶之路

https://tobebetterjavaer.com/java8/optional.html

**Comment by xiaoacaia** [comment]
04、判断值是否存在 中 Optional<String> optOrNull = Optional.ofNullable(null); 
 System.out.println(opt.isPresent()); // 输出：false 中，输出的 opt 应改为 optOrNull,并且 Java 11 后还可以通过方法 isEmpty()下面的输出还是 isPresent
